server/routerlicious/api-report/server-services-client.api.md

```diff
@@ -484,7 +485,11 @@ export class NetworkError extends Error {
     readonly code: number;
     get details(): INetworkErrorDetails | string;
     readonly isFatal?: boolean;
+    readonly retryAfter: number;
     readonly retryAfterMs?: number;
+    toJSON(): INetworkErrorDetails & {
```

Comment:
The client models the errors coming over the socket as `IR11sSocketError`, which doesn't have `isFatal`.  We assume all non-retryable errors are fatal - where would `canRetry` and `isFatal` both be false and what should the client do?  If that's an important case we should open an issue to update `IR11sSocketError` to include `isFatal` so we can properly handle it.

Reply:
canRetry and isFatal are both used internally within R11s (primarily by Scribe) and are ignored by the client. The driver currently does not parse the response body for information like this and is primarily noise. As I was going through this PR, I wanted to find a way to change it so that external request errors only send `message` but internal request errors contain this extra info. Unfortunately, most public APIs are also called internally, so it is not trivial :/ I can open an issue to track this

Reply:
Opened #9910 and have asked the FRS team to work with me to increase priority of improving our CI confidence